doc/source/notebooks/advanced/mcmc.ipynb

```diff
@@ -968,7 +967,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We then run the sampler,"
+    "We then run the sampler."
```

Comment:
This should be a comma, followed by the code running the sampler... @awav @condnsdmatters  ??